plantcv/plantcv/threshold/threshold_methods.py

```diff
@@ -265,6 +265,59 @@ def triangle(gray_img, max_value, object_type="light", xstep=1):
     return bin_img
 
 
+def texture(gray_img, kernel, threshold, offset=3, texture_method='dissimilarity', borders='nearest',
+            max_value=255):
+    """Creates a binary image from a grayscale image using skimage texture calculation for thresholding.
+    This function is quite slow.
+
+    Inputs:
+    gray_img       = Grayscale image data
+    kernel         = Kernel size for texture measure calculation
+    threshold      = Threshold value (0-255)
+    offset         = Distance offsets
+    texture_method = Feature of a grey level co-occurrence matrix, either
+                     'contrast', 'dissimilarity', 'homogeneity', 'ASM', 'energy',
+                     or 'correlation'.For equations of different features see
+                     scikit-image.
+    borders        = How the array borders are handled, either 'reflect',
+                     'constant', 'nearest', 'mirror', or 'wrap'
+    max_value      = Value to apply above threshold (usually 255 = white)
+
+
+    Returns:
+    bin_img        = Thresholded, binary image
+
+    :param gray_img: numpy.ndarray
+    :param kernel: int
+    :param threshold: int
+    :param offset: int
+    :param texture_method: str
+    :param borders: str
+    :param max_value: int
+    :return bin_img: numpy.ndarray
+    """
+    from skimage.feature import greycomatrix, greycoprops
```

Comment:
The two import statements can go at the top of the file with the other imports. From what I have read it doesn't sound like there's much speed benefit to putting the imports inside a function, unless you want to limit the namespace scope (like we do with matplotlib). Otherwise stylistically the Python preference is at the top of the module.